### ISSUE TITLE
Accessibility - Add aria-hidden on activity images

### ISSUE THREAD

**Issue by slafleche** [vanilla/vanilla#5948]
For activity page, set aria-hidden="true" for images with no alt text.

**Comment by slafleche** [comment]
In PR: https://github.com/vanilla/vanilla/pull/5978
